Marlin/src/module/delta.cpp

```diff
@@ -278,6 +280,11 @@ bool home_delta() {
   HOMEAXIS(B);
   HOMEAXIS(C);
 
+  // Re-enable stealthChop if used. Disable diag1 pin on driver.
+  #if ENABLED(SENSORLESS_HOMING)
+    delta_sensorless_homing(false);
+  #endif
+
```

Comment:
@thinkyhead 
The shift of ` delta_sensorless_homing(false)` behind of the `HOMMEAXIS()` calls feels wrong to me. 
The code blocks bracketed by `sensorless_homing_per_axis();` and `sensorless_homing_per_axis(false);` are nested now. 
`HOMEAXIS()` is calling `do_homing_move()` several times, where the sensorless homing is currently switched on and off.

The other problem is calling `do_homing_move()` in `HOMEAXIS()` also for the moves away from the endstops. Maybe enabling/disabling sensorless homing should be done at the relevant places in `HOMEAXIS()` only.